ramlwrap/utils/validation.py

```diff
@@ -41,10 +41,17 @@ def add_action(self, request_method, action):
         :param action: the action to map to the request.
         :returns: returns nothing.
         """
+
+        # if dynamic values, replace the placeholders in url with the dynamic value regex
+        if action.regex:
```

Comment:
I'm not sure this is quite the right place to do this... it seems a bit Ott as we're going to override the url regex for *every* action that we add to it. We just want to do this once so we could either do it in the raml.py or do it as part of the local_endpoint init?